src/prefect_server/api/__init__.py

```diff
@@ -8,3 +8,5 @@
 import prefect_server.api.runs
 import prefect_server.api.messages
 import prefect_server.api.agents
+import prefect_server.api.artifacts
+import prefect_server.api.deprecated
```

Comment:
what's this `prefect_server.api.deprecated` for?

Reply:
Oh I think that's a merge conflict accidental add, will remove